addons/sale/report/ir_actions_report_templates.xml

```diff
@@ -80,7 +81,7 @@
                         <th name="th_discount" t-if="display_discount" class="text-end">
                             <span>Disc.%</span>
                         </th>
-                        <th name="th_taxes" class="text-end">Taxes</th>
+                        <th name="th_taxes" t-attf-class="text-end {{ 'd-none' if not show_taxes else '' }}">Taxes</th>
```

Comment:
You will have to adapt the templates of Avatax as well

Note that the "over-saturation" should be resolved in this module...

Reply:
Sorry, but What you mean by this term please `over-saturation`?

Reply:
> Reason: in some scenarios users can see several taxes listed in the "Taxes" column in the PDF report (for SO, quotes, invoices, etc).
**Having the column <ins>oversaturated</ins> with all the taxes can represent a problem for our users.**

Reply:
@william-andre 
any update regarding this comment?
Thanks!

Reply:
I don't understand your comment.
However, there is this thing
```xml
    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
        <!-- TODO not the best thing to inherit... -->
        <xpath expr="//span[@id='line_tax_ids']/.." position="attributes">
            <attribute name="t-if">not o.is_avatax</attribute>
        </xpath>
        <xpath expr="//th[@name='th_taxes']" position="attributes">
            <attribute name="t-if">not o.is_avatax</attribute>
        </xpath>
    </template>
```

And AFAIK, it should already solve the goal of this PR